### DISCUSSION TITLE
Need instruction after step17 and internet issue

### DISCUSSION THREAD

**Discussion by galosre** [tesla-android/issue-tracker#17] [General]
From a computer got 
the desktop android 12,  but I cannot click on anything, Maybe I need Bluetooth connectivity from the car.

Anyway the main issue I got no internet therefore not able to connect to the car's Wi-Fi with LTE inserted in any of the RPI USB
 I double-check the IK41UC is working from a Mac with Wi-Fi off, got good internet speed (and by the way the instructions  from Alcatel for IK41UC are wrong about LED colors you can check you've got LTE at 192.168.1.1)
How to connect the CarlinKit CPC200-Autokit, CPC200-CCPA?
![screen](https://user-images.githubusercontent.com/22267106/168120120-28a59207-5dca-4f41-ae08-69e281bbd51a.jpg)

**Comment by alternateal** [comment]
How to connect the CarlinKit CPC200-Autokit, CPC200-CCPA?

Plug the CPC200 into the Pi running Android and then run autokit.apk from the CPC200 and let it update itself across the internet.

**Comment by mbhforum** [comment]
I am slighly confused.  It says the CPC200 is optional for Wireless Carplay.  Do you need it for Wired Carplay too? If so, why is it listed as optional?

**Reply by mikegapinski** [reply]
Optional meaning no CPC200 no CarPlay at all

**Reply by mbhforum** [reply]
Thanks! I recommend mentioning this as the current documentation led me to believe that adapter is only necessary if you wanted wireless CarPlay and that wired CarPlay was built in to the solution.  I would also venture that the majority of people are doing this for CarPlay or Android Auto and not just to run Android in their cars.
